open-telemetry/otel-getting-started.mdx

```diff
@@ -0,0 +1,145 @@
+  url: "https://sandbox.middleware.io:443",
+});
+
+// Set exporter to use Middleware gRPC metric endpoint
+const metricExporter = new OTLPMetricExporter({
+    url: "https://sandbox.middleware.io:443",
+});
+
+// Set exporter to use Middleware gRPC trace endpoint
+const traceExporter = new OTLPTraceExporter({
+    url: "https://sandbox.middleware.io:443",
+});
+
+```
+
+```javascript OTLP/HTTP
+// Send HTTP POST requests to endpoints:
+POST https://<UID>.middleware.io/v1/logs
+POST https://<UID>.middleware.io/v1/traces
+POST https://<UID>.middleware.io/v1/metrics
+```
+</CodeGroup>
+
+<Accordion title="Sample OTLP/HTTP Curl Request">
+
+The following is an example of sending metrics data via curl request.
+
+<Note> We do not recommend using this specific pattern for OTLP/HTTP curl requests. Data must be in OTLP/HTTP format. Learn more [here](https://opentelemetry.io/docs/specs/otlp/#otlphttp). </Note>
+```
+curl -X POST "https://demo.middleware.io/v1/metrics" \
+-H "Accept: application.json" \
+-H "Content-type: application.json" \
+-d @ << EOF
+{
+    "resource_metrics": [
+        {
+            "resource": {
+                "attributes": [
+                    {
+                        "key": "mw.account_key",
+                        "value": {
+                            "string_value": "xxxxxxxxxx"
+                        }
+                    },
+                    {
+                        "key": "mw.resource_type",
+                        "value": {
+                            "string_value": "custom"
+                        }
+                    }
+                ]
+            },
+            "scope_metrics": [
+                {
+                    "metrics": [
+                        {
+                            "name": "swap-usage",
+                            "description": "SWAP Usage",
+                            "unit": "bytes",
+                            "gauge": {
+                                "data_points": [
+                                    {
+                                        "attributes": [
+                                            {
+                                                "key": "device",
+                                                "value": {
+                                                    "string_value": "nvme0n1p4"
+                                                }
+                                            }
+                                        ],
+                                        "start_time_unix_nano": 1673435153000000000,
+                                        "time_unix_nano": 1673435153000000000,
+                                        "asInt": 400500678
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+}
+EOF  
+```
+</Accordion>
+
+### Step 3: Send Data to Middleware
```

Comment:
This is worded such that I expect some more instructions. Does setting up the above exporters automatically start sending Middleware data, or is there other work the user has to do?

Reply:
I think that this is the full extent of the basic set up. I will remove step 3 since it's unnecessary.